.gitignore

```diff
@@ -9,3 +9,4 @@ data/wiki_specialphrases.sql
 data/osmosischange.osc
 
 .vagrant
+data/country_osm_grid.sql.gz
```

Comment:
As you suggested I have add it to the .gitignore, I hope it doesn't make any trouble if it is in this history.